app/javascript/article-form/components/Form.jsx

```diff
@@ -18,12 +18,16 @@ export const Form = ({
   switchHelpContext,
   errors,
 }) => {
+  // 🚑 Detect if user is using chromium to disable grammarly
+  const isChrome = !!window.chrome;
```

Comment:
This check will also return true for all Chromium-based browsers which we don't want as Grammarly currently works in at least Edge (Chromium). We could combine this check with a regex test on the user-agent, e.g.

`!/(Chrome\/.+Edg\/)|/.test(navigator.userAgent)`

I haven't checked the Brave user-agent or others, but we could add them in the regex.

Reply:
I think that regex may be incorrect

[![image](https://user-images.githubusercontent.com/3534427/119118335-03c79d00-ba22-11eb-8fb5-20cc24bf37d2.png)](https://regexper.com/#%2F%28Chrome%5C%2F.%2BEdg%5C%2F%29%7C%2F)

Reply:
Hmm I tested it yesterday in Chrome, Edge, and Firefox and it seem to behave properly. I'm using `.+` in the middle as there is a lot in the user-agent string. I can help with this if it's blocking you.